src/utils/violations/tabSwitch.js

```diff
@@ -1,11 +1,10 @@
 import { VIOLATIONS } from '../constants';
 import { getIsBrowserBlurred } from './browserBlur';
 
-let visibilityChangeHandler = null;
-
 export default function detectTabSwitch(handleViolation) {
-  visibilityChangeHandler = () => {
-    if (document.hidden && !getIsBrowserBlurred()) {
+  const visibilityChangeHandler = () => {
+    const isBlurred = getIsBrowserBlurred();
+    if (document.hidden && !!isBlurred) {
```

Comment:
Why do we need this blur check ? Wht is the reason behind adding this check ? Add this in the PR Description.